### DISCUSSION TITLE
`docker push`支持断点续传 Interruption Resuming for `docker push`

### DISCUSSION THREAD

**Discussion by hero-intelligent** [moby/moby#46373] [Polls]
I've noticed that issues like #4872 has been opened multiple times, and then be closed friendly but ruthlessly anyways in a robotic pattern like `Thanks for reporting! This is a duplicate of https://github.com/moby/moby/issues/4872, you can subscribe to that issue to stay informed on updates`. I have opened an issue, and I clearly know that my issue will be closed as well, so I opened a discussion here.

I just want to ask the developers in this project: is there anyone in the program still making any progress on this work at all?  If you want to add this feature in this project, JUST DO IT and give the community a promise; or otherwise, REJECT DIRECTLY WITH PROPER REASON WHY YOU WON'T FIX IT, instead of replying in words in a robotic pattern like `Thanks for reporting! This is a duplicate of https://github.com/moby/moby/issues/4872, you can subscribe to that issue to stay informed on updates`.

**Comment by hero-intelligent** [comment]
中文翻译：

我注意到像 #4872 这样的issue已经开了很多次，然后都看似友好但无情地关闭了，并附上机器人式回复：“感谢报告！这是 https://github.com/moby/moby/issues/4872 的副本，您可以订阅该问题以随时了解更新“。我也开了一个issue，我也清楚地知道我的问题也将被关闭，所以我在这里打开了一个讨论。

我只想问这个项目的开发人员：项目中到底还有没有人在这项工作上取得任何进展？如果确实你想在这个项目中添加这个功能，踏实干，并向社区做出承诺；否则，直接拒绝，并附上适当的理由说明为什么您不会修复它，而不是像机器人一样回复，“感谢您的报告！”这是 https://github.com/moby/moby/issues/4872 的副本，您可以订阅该问题以随时了解更新。“

**Comment by hero-intelligent** [comment]
It has been 9 YEARS when the first issue was opened. I was surprised that there is no other similar discussions here, but all closed issues. I'm wondering why they won't fix that issue for such a long time regardless of so many people's needs.
已经过去了9年了，问题还没解决。我很惊讶的是，为什么只有被关闭的issue，却没有人在这里提过discussion，我更想知道这么长时间过去了，为什么这么多人需要的功能直到现在还没能解决。

**Comment by neersighted** [comment]
https://github.com/moby/moby/issues/46372#issuecomment-1699284364
